test/screenshot/floating-label/index.js

```diff
@@ -0,0 +1,83 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import FloatingLabel from '../../../packages/floating-label';
+
+import '../../../packages/floating-label/index.scss';
+import './index.scss';
+
+class Input extends React.Component {
+  state = {
+    shouldUsePrimaryLabel: true,
+    shouldFloat: false,
+    width: 0,
+  };
+
+  constructor(props) {
+    super(props);
+    this.floatingLabelElement_ = React.createRef();
+  }
+
+  render() {
+    const {shouldUsePrimaryLabel, shouldFloat} = this.state;
+    const labelText = ['My Label Text', 'Alternate Label Text'];
+
+    return (
+      <div>
+        <div className='input__container'>
+          <div className='input__dock'>
+            <FloatingLabel
+              shouldFloat={shouldFloat}
+              ref={this.floatingLabelElement_}
+              setWidth={(width) => this.setState({width})}
+            >
+              {shouldUsePrimaryLabel ? labelText[0] : labelText[1]}
+            </FloatingLabel>
+          </div>
+          {this.renderControls()}
+        </div>
+      </div>
+    );
+  }
+
+  renderControls() {
```

Comment:
instead of having a button to test float, just have multiple instances of FloatingLabel on the same screenshot page.